src/components/KeyboardCompatibleView.js

```diff
@@ -42,7 +42,7 @@ export class KeyboardCompatibleView extends React.PureComponent {
     super(props);
 
     this.state = {
-      channelHeight: new Animated.Value('100%'),
+      channelHeight: new Animated.Value(Dimensions.get('window').height),
```

Comment:
the type Animated.Value expects is number so using the string 100% wasn't valid before